src/utils/logger.ts

```diff
@@ -1,23 +1,26 @@
+import winston, { format } from 'winston';
+
+const { combine, colorize, simple } = format;
+
 export enum LogLevel {
-  debug = 1,
-  info = 2,
-  warn = 3,
-  error = 4,
+  error = 'error',
```

Comment:
I changed log levels to text in order to avoid switching by log level in ConsoleLogger